tests/test_shell.py

```diff
@@ -231,9 +236,9 @@ def test_shell_with_jobs_and_selection(self):
         for i in range(3):
             project.open_job(dict(a=i)).init()
         assert len(project)
+        python_command = 'python -m signac shell -f a.{}gt 0'.format('$' if WINDOWS else r'\$')
```

Comment:
This is hacky but I couldn't figure out a better way that uses Python's standard library. On Windows, `$` won't work with the backslash to escape it. However, POSIX requires the backslash.

Reply:
It is for a test, so I wouldn't be too worried about it being 'hacky'. I don't think it is too bad though.